kolibri/logger/serializers.py

```diff
@@ -51,6 +54,8 @@ def get_totalattempts(self, obj):
         return AttemptLog.objects.filter(masterylog__summarylog=obj.summarylog).count()
 
 class AttemptLogSerializer(serializers.ModelSerializer):
+    answer = serializers.JSONField(default='{}')
```

Comment:
Unfortunately, it seems that the third party JSON Field, and DRF JSONField do not cooperate that well, so we end up having to duplicate these defaults here.